resolwe_bio/tools/make_report.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python2
+"""Generate amplicon report."""
+import argparse
+import subprocess
+
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+
+DECIMALS = 3  # Decimal precision when presenting results:
+
+# Set defult colors:
+GENIALIS_BLUE = '#00bcd4'
+GENIALIS_GREY = '#9E9E9E'
+mpl.rcParams['axes.labelcolor'] = GENIALIS_GREY
+mpl.rcParams['axes.edgecolor'] = GENIALIS_GREY
+mpl.rcParams['lines.color'] = GENIALIS_GREY
+mpl.rcParams['lines.antialiased'] = False
+mpl.rcParams['text.color'] = GENIALIS_GREY
+mpl.rcParams['xtick.color'] = GENIALIS_GREY
+mpl.rcParams['ytick.color'] = GENIALIS_GREY
+
+parser = argparse.ArgumentParser(description='Fill data into tex template file.')
```

Comment:
We typically follow the convention to use double quotes for strings meant for human consumption and use single quotes for everything else. Please, use double quotes for the description and all help strings.